pkg/apis/dashboard/migration/schemaversion/v38.go

```diff
@@ -0,0 +1,122 @@
+package schemaversion
+
+// V38 updates the configuration of the table panel to use the new cellOptions format
+// and updates the overrides to use the new cellOptions format
+func V38(dashboard map[string]interface{}) error {
```

Comment:
I wonder, when such a migration function would throw an error? Since we are continuing on every !ok when asserting type, it seems like this function can never technically error.

Reply:
is that not ok? If the migration cannot be performed, it should be moved to the next, right? We should make the best effort for each of the migrations. Do you see any case where it should be thrown?

Reply:
no, you are right - it's just why there's an `error` as a return type, when it never returns an error :)

Reply:
Ups, you're right, I didn't catch that! Fixing it! thanks!

Reply:
my assumption was that it might be useful when debugging to return errors when migrations cannot continue. that way the caller can know the status and decide what to do when a migration fails, even if that means ignore it and/or log it. this can also be useful in unit testing as well because it's easier to check those edge cases. thoughts?

Reply:
I'm fine with that @toddtreece, i see your point